src/Components/NavbarMain.js

```diff
@@ -3,7 +3,7 @@ import Container from 'react-bootstrap/Container';
 import Nav from 'react-bootstrap/Nav';
 import Navbar from 'react-bootstrap/Navbar';
 import { NavLink } from 'react-router-dom';
-import logo67x40 from './lingo-bingo-logo-bravo-67-40.png';
+// import logo67x40 from 'public/lingo-bingo-logo-bravo-67-40.png';
```

Comment:
Is this commented out permanently? If so double-check that the image shows as expected and then delete the line.

Reply:
Yes, image files in `public` to be loaded using the `src` attribute, because we cannot use import statements to access anything outside of `src`. I will delete the lingering comments, good catch!